django_cte/cte.py

```diff
@@ -45,6 +45,7 @@ class CTE:
     """
 
     def __init__(self, queryset, name="cte", materialized=False):
+        self.qs = queryset
         self.query = None if queryset is None else queryset.query
```

Comment:
Is this still needed in addition to the new queryset reference? Seems like probably not.